share/xtp/xml/neighborlist.xml

```diff
@@ -1,6 +1,11 @@
 <?xml version="1.0"?>
 <options>
   <neighborlist help="Determines neighbouring pairs for exciton transport" section="sec:neighborlist">
-    <constant help="cutoff for all other types of pairs" unit="nm" default="1.5" choices="float+"/>
+    <type help="segments names if the segmentname method is used"/>
```

Comment:
```suggestion
    <segmentname help="segments names if the segmentname method is used"/>
```

Reply:
@baumeier I am introducing this change, just to let you know because you are using this feature